src/WebJobs.Script.WebHost/App_Start/WebApiConfig.cs

```diff
@@ -93,6 +93,7 @@ private static WebHostSettings GetDefaultSettings(ScriptSettingsManager settings
                 settings.ScriptPath = settingsManager.GetSetting(EnvironmentSettingNames.AzureWebJobsScriptRoot);
                 settings.LogPath = Path.Combine(Path.GetTempPath(), @"Functions");
                 settings.SecretsPath = HttpContext.Current.Server.MapPath("~/App_Data/Secrets");
+                settings.IsSelfHost = true;
```

Comment:
This overloads `IsSelfHost` which is currently used to indicate when self hosting in a console app. Local runs in IIS/IIS Express should continue to have this return `false`